node/src/main/scala/coop/rchain/node/grpc.scala

```diff
@@ -59,7 +59,10 @@ object GrpcServer {
   class ReplImpl(runtime: Runtime) extends ReplGrpc.Repl {
     import RholangCLI._
     // TODO we need to handle this better
-    import monix.execution.Scheduler.Implicits.global
+    import monix.execution.Scheduler
+    import monix.execution.schedulers.SchedulerService
+
+    implicit val io: SchedulerService = Scheduler.io("repl-io")
```

Comment:
isn't the `global` scheduler of the same type as one created with `Scheduler.io` ? I'm curious why this is needed. All threads were taken a new ones weren't created?

Reply:
No, it uses ForkJoinPool. It was eating up resources on VM environments.